src/components/PeoplePages/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,238 @@
+import { useCallback, useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import classnames from 'classnames';
+import debounce from 'lodash/debounce';
```

Comment:
`'lodash' should be listed in the project's dependencies. Run 'npm i -S lodash' to add it`